RecoEgamma/EgammaTools/interface/EnergyScaleCorrection.h

```diff
@@ -81,7 +81,7 @@ class EnergyScaleCorrection {
   public:
     CorrectionCategory(const std::string& category, int runnrMin = 0, int runnrMax = 999999);
     CorrectionCategory(
-        const unsigned int runnr, const float et, const float eta, const float r9, const unsigned int gainSeed)
+        const unsigned int runnr, const double et, const double eta, const double r9, const unsigned int gainSeed)
```

Comment:
A general question, why all the changes from floats to doubles?

Reply:
I saw problems at boundaries of the corrections (e.g. eta=2.5000001 which was at a precision of 10^-7). So I had to update from float to double.